src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java

```diff
@@ -29,4 +29,6 @@ public void fooMethod() {
         1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
         13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
       };
+
+  PackageStatementTest packageStatementTest = new PackageStatementTest(); // ok
```

Comment:
created instance in this file also, just to maintain similarity between both input files

Reply:
done